src/lib.rs

```diff
@@ -556,6 +570,14 @@ impl Wizer {
         self
     }
 
+    /// Enable or disable the Wasm reference-types proposal.
+    ///
+    /// Defaults to `false`.
```

Comment:
Can you also add the extra disclaimer paragraph that the CLI flag docs has here too? Unfortunate that we have this duplication, but I don't think there is a better option, unfortunately.